webpackjs/components/DocklessData/index.js

```diff
@@ -130,54 +104,36 @@ class DocklessData extends Component {
       });
   }
 
-  getValue(data, month, year, metric) {
-    let leData = `${data}`;
-    const monthYear = `${month}_${year}`;
+  componentDidMount() {
+    this.runQueries(this.state.monthYear);
+  }
 
-    if (!this.state[leData]) {
-      return 0;
-    }
+  getValue(data, metric) {
+    const leData = `${data}`;
 
-    if (!this.state[leData][monthYear]) {
-      return "no data";
+    // return 0 when the API hasn't responded yet but the HTML needs to render
+    if (!this.state[leData] || this.state.dataIsLoaded === false) {
+      return 0;
```

Comment:
I see what you're doing here, and why you moved away from using the string "no data". I feel like it works. I'm just wondering if a spinner, or "loading" string might be more accurate. Just a thought to play around with in another PR if you feel strongly about it.

The only place it matters is 311 calls pre June 2018. It is not necessarily true that there were 0 311 calls about scooters in those first two months, 311 just didn't establish a classification for those types of calls yet. Very minor difference that probably doesn't matter honestly. [carry on]